app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -130,6 +131,19 @@ public function panel(Panel $panel): Panel
             ])
             ->authMiddleware([
                 Authenticate::class,
+            ])
+            ->plugins([
+                BreezyCore::make()
+                    // See https://github.com/jeffgreco13/filament-breezy
+                    ->myProfile(
+                        shouldRegisterUserMenu: true,
+                        shouldRegisterNavigation: false,
+                        hasAvatars: false,
+                        slug: 'my-profile',
+                    )
+                    ->enableTwoFactorAuthentication(
+                        force: false,
```

Comment:
if this is to enforce 2fa, then it's best to enable that by default but add a `.env` entry `2FA_ENABLED` to allow overriding it for development purposes